masterlist.yaml

```
@@ -4887,6 +4887,18 @@ plugins:
       - Graphics
       - Stats
 
+  - name: 'Torch Brightness Radius.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/5101/' ]
```

You can add a name to locations that LOOT can then use when displaying them:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/5101/'
        name: 'Torch Brightness Radius'
```